lib/tasks/dev.rake

```diff
@@ -101,22 +103,45 @@ namespace :dev do
         if is_established
           unless raw_enrollment_status == 'pending' && !user.pending_in_person_enrollment.nil?
             profile = Profile.new(user: user)
+            usps_compatible_number_alternative = n.to_s.chars.map do |c|
+              ('a'.ord + c.to_i).chr
+            end.join('')
             pii = Pii::Attributes.new_from_hash(
               first_name: 'Test',
-              last_name: "User #{n}",
+              last_name: "User #{usps_compatible_number_alternative}",
               dob: '1970-05-01',
               ssn: "666-#{n}", # doesn't need to be legit 9 digits, just unique
```

Comment:
Do we need to worry about duplicates across multiple runs?

Reply:
Duplicates are expected, and that's fine.